test/e2e/db_users_oidc_test.go

```diff
@@ -73,8 +73,8 @@ var _ = Describe("Operator to run db-user with the OIDC feature flags", Ordered,
 					Namespace: config.DefaultOperatorNS,
 					Name:      config.DefaultOperatorGlobalKey,
 				},
-				Namespace:    config.DefaultOperatorNS,
-				FeatureFlags: featureflags.NewFeatureFlags(func() []string { return []string{} }),
+				WatchedNamespaces: map[string]bool{config.DefaultOperatorNS: true},
```

Comment:
I believe in my watch namespaces fix, this is gone. But it is not important, we will merge sooner or later.

Reply:
Not really, this is a namespaced test